applications/luci-app-openwisp/Makefile

```diff
@@ -0,0 +1,15 @@
+# Copyright 2021 Nicholas Smith (nicholas@nbembedded.com)
+# This is free software, licensed under the GNU General Public License v2.
+
+include $(TOPDIR)/rules.mk
+
+PKG_LICENSE:=GPL-2.0-or-later
+PKG_MAINTAINER:=Nicholas Smith <nicholas@nbembedded.com>
+
+LUCI_TITLE:=LuCI support for OpenWISP
+LUCI_DESCRIPTION:=Allows configuration of OpenWISP agent settings
+LUCI_DEPENDS:=+luci-mod-admin-full +openwisp-config
```

Comment:
since we have a javascript module here and not a lua module i would say we don't need the dependency
`+luci-mod-admin-full`

Reply:
Thanks @feckert , that has been removed